tc/sgx/trusted_worker_manager/enclave/kme_workload/kme_workload_plug-in.cpp

```diff
@@ -310,16 +335,16 @@ void KMEWorkloadProcessor::ProcessWorkOrder(
     const ByteArray& work_order_id,
     const std::vector<tcf::WorkOrderData>& in_work_order_data,
     std::vector<tcf::WorkOrderData>& out_work_order_data) {
-    ExtWorkOrderInfoKME* ext_wo_info_kme = \	
-        (ExtWorkOrderInfoKME*) ext_work_order_info; 
+
+    ext_wo_info_kme = (ExtWorkOrderInfoKME*) ext_work_order_info;
```

Comment:
`ext_wo_info_kme` is unused in this function

Reply:
ext_wo_info_kme is a private member of KMEWorkloadProcessor class. Its initialized here.
This avoids passing the instance to functions like Register, PreProcessWorkOrder..

Reply:
Please use dynamic_cast and move this casting to respective functions where `ExtWorkOrderInfoKME` is getting used

Reply:
Done